<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>401</height>
   </rect>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>20</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>db2xls and xls2db</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>360</y>
     <width>89</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>216</y>
     <width>537</width>
     <height>129</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>2. Excel表格导出为.db件或.sjf文件</string>
   </property>
   <widget class="QPushButton" name="pushButton_xls2db">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>88</y>
      <width>97</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
     </font>
    </property>
    <property name="text">
     <string>导出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_txt">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>24</y>
      <width>169</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>以sheetname为数据表名字</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_selecxls">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>56</y>
      <width>97</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
     </font>
    </property>
    <property name="text">
     <string>选择Excel文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_xlspath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>56</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="text">
     <string>xls文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>88</y>
     <width>537</width>
     <height>105</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="title">
    <string>1. db文件或sjf文件导出为Excel表格</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_dbpath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>32</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="text">
     <string>db文件、sjf文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_selecdb">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>33</y>
      <width>97</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
     </font>
    </property>
    <property name="text">
     <string>选择数据库文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_db2xls">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>64</y>
      <width>97</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
     </font>
    </property>
    <property name="text">
     <string>导出</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_quit</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_quit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
